lib/src/signed_video_h26x_auth.c

```diff
@@ -1025,18 +1023,6 @@ maybe_validate_gop(signed_video_t *self, h26x_nalu_t *nalu)
       // validation in the middle of a stream. Now it is time to reset it.
       validation_flags->is_first_validation = !validation_flags->signing_present;
 
-      if (validation_flags->reset_first_validation) {
-        validation_flags->is_first_validation = true;
```

Comment:
Should you really remove the reset? This is used to handle the case where there are more than one SEI in the beginning of a stream that cannot be used. The scrapped test for example.

Reply:
But as I understand its for the first validation and  resetting  the first validation is done without a flag in h26x_nalu_list_remove_missing_items function.

Reply:
Hmm, I'm not 100% convinced. Here is an example
If `IPIPPIPPPIPPPPIP`
is signed into
`IPIPPISPSPPISPPPPISP`
and remove the first 2 GOPs to
`IsPsPPISPPPPISP`
then the first 2 `s` do not belong to this recording. In this case the first 2 validations need to be discarded.
Make a test of this